src/libraries/Microsoft.Extensions.Hosting/src/HostApplicationBuilder.cs

```diff
@@ -7,6 +7,7 @@
 using System.IO;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Diagnostics.Metrics;
```

Comment:
I'm not well versed in our various hosting APIs so its probably good to get someone with more expertise in this area to take a look once you are ready.